subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/http/DeprecatedTLSVersionDependencyResolutionIntegrationTest.groovy

```diff
@@ -76,27 +107,27 @@ class DeprecatedTLSVersionDependencyResolutionIntegrationTest extends AbstractHt
         when:
         module.allowAll()
         and:
-        executer.withArgument("-Dhttps.protocols=TLSv1,TLSv1.1")
+        executer.withArgument("-Dhttps.protocols=TLSv1.2")
```

Comment:
🤔 would it be better to just disable this tets on the lastest JDK? Or have a test that maintains this logic on the older versions of the JDK but have one variation for the latest version?